render/egl.c

```diff
@@ -77,6 +76,16 @@ static bool check_egl_ext(const char *exts, const char *ext) {
 	return false;
 }
 
+static bool load_egl_proc(void *proc_ptr, const char *name) {
+	void *proc = (void *)eglGetProcAddress(name);
+	if (proc == NULL) {
+		wlr_log(WLR_ERROR, "eglGetProcAddress(%s) failed", name);
+		return false;
```

Comment:
I wonder if it's worth replacing this with `abort()`.
A driver failing to give us a function after advertising the extension sounds like a serious driver bug, or our bug because we spelt the function name wrong.

I don't think this is worth trying to recover from, and greatly simplifies some of the conditions later.
Weston doesn't even bother to check the return, probably for the same reasons.

Reply:
Yeah, I agree. Done.